build/metaschema/json-binding.md

```diff
@@ -0,0 +1,352 @@
+
```

Comment:
This file was used to define the work needed for this PR. I don't think this needs to be added to the repo as a standalone Markdown file. We should pull from this to build out metaschema documentation and examples instead.

Reply:
Moved the file to a safe place, no longer in the rep